server/middlewares/authorization.js

```diff
@@ -42,6 +42,24 @@ class authorization {
     request.isAuthorized = userId === request.userId; // BOOLEAN
     return next();
   }
+
+  /**
+   * This checks whether the requester (user) is authorized
+   * to complete the request on an article.
+   * @param {object} request
+   * @param {object} response
+   * @param {function} next
+   * @returns {function} callback function
+   */
+  static async article(request, response, next) {
```

Comment:
article is definetely not a good name for this middleware

Reply:
You are right @seyi-adeleke. `patch` will be more appropriate here.